test/test_vtk_export.jl

```diff
@@ -41,4 +41,22 @@
             @test bytes2hex(open(SHA.sha1, fname*".vtu")) == "31b506bd9729b11992f8bcb79a2191eb65d223bf"
         end
     end
+    @testset "write_cellset" begin
+        # More tests in `test_grid_dofhandler_vtk.jl`, this just validates writing all sets in the grid
+        # which is not tested there, see https://github.com/Ferrite-FEM/Ferrite.jl/pull/948
+        mktempdir() do tmp
+            grid = generate_grid(Quadrilateral, (2,2))
+            addcellset!(grid, "set1", 1:2)
+            addcellset!(grid, "set2", 1:4)
+            manual = joinpath(tmp, "manual")
+            auto = joinpath(tmp, "auto")
+            VTKFile(manual, grid) do vtk
+                Ferrite.write_cellset(vtk, grid, keys(Ferrite.getcellsets(grid)))
```

Comment:
I'm not sure if this is the best test, since the order is not part of the API, and while this test works now, it would fail if we decide to e.g. sort the keys in `write_cellset`. 
I'd be happy to hear if someone has ideas for a better test!